### PR TITLE
Update Twilio Settings

### PR DESCRIPTION
**Pull request by ericnewcomer**
* Moves viewing and removing of Twilio credentials to account page with ability to edit them
* Bump code coverage in channels/views.py

### CODE REVIEW

File: temba/channels/tests.py [hunk 1]
```diff
@@ -92,17 +93,58 @@ def test_message_context(self):
...
+        self.assertEquals(404, response.status_code)
+
     def test_delegate_channels(self):
 
+        self.login(self.admin)
+
         # we don't support IVR yet
         self.assertFalse(self.org.supports_ivr())
 
+        # pretend we are connected to twiliko
```
Comment (nicpottier) [inline temba/channels/tests.py]: is a twiliko a small twilio?

Comment (ericnewcomer) [inline temba/channels/tests.py]: ya, its much cuter too.

**Comment by nicpottier** [comment]
So how do you get to the twilio page?

**Comment by ericnewcomer** [comment]
If you have a Twilio Account connected, it'll show up on your account page with the option to edit it.

**Comment by nicpottier** [comment]
Nice, looks good.

**Pull request merged**